src/core/layout/qgslayoutitemregistry.h

```diff
@@ -428,6 +428,26 @@ class CORE_EXPORT QgsLayoutItemRegistry : public QObject
      */
     bool addLayoutItemType( QgsLayoutItemAbstractMetadata *metadata SIP_TRANSFER );
 
+    /**
+     * Unregisters a layout item type
+     */
+    bool removeLayoutItemType( int typeId );
+
+    /**
+     * Unregisters a layout item type
```

Comment:
```suggestion
     * Unregisters a layout item type.
     *
     * The \a metadata object will be deleted and should not be used after this call.
     *
     * \since QGIS 4.0
```